troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -56,22 +62,26 @@ export default React.createClass({
             || username === image.get("created_by").username
             || profile.get("is_staff")) {
             return (
-            <div className="edit-link-row">
-                <a className="edit-link" onClick={this.onEditClicked}><span className="glyphicon glyphicon-pencil" /> Edit Version</a>
+            <div>
+                <a onClick={this.onEditClicked}>
+                    <span className="glyphicon glyphicon-pencil" />
+                    { " Edit Version" }
```

Comment:
Use padding not a string space here. I don't like using jsx containers just for strings.